specs/windows/bam.table

```diff
@@ -0,0 +1,11 @@
+table_name("bam")

```

Comment:
IMO, we should call this `background_activities_moderator` unless the abbreviation `bam` is well known and commonl used.